src/auth/computeclient.ts

```diff
@@ -26,10 +29,17 @@ export interface ComputeOptions extends RefreshOptions {
    * may have multiple service accounts.
    */
   serviceAccountEmail?: string;
+  /**
+   * The scopes that will be requested when acquiring service account
+   * credentials. Only applicable to modern App Engine and Cloud Function
+   * runtimes as of March 2019.
+   */
+  scopes?: string|string[];
```

Comment:
Does anything break if we use just `string[]` here? Are there any usages of `string`?

Reply:
I expect that users will want to just say `getTokenWithScope('https://...')` rather than having to wrap it `getTokenWithScope(['https://...'])`.

Reply:
Ah, I didn't realize it's an exported interface. OK!